vars/main.yml

```diff
@@ -59,3 +59,4 @@ __mssql_ha_private_key_dest: /var/opt/mssql/data/{{ mssql_ha_cert_name }}.pvk
 
 __mssql_ad_kinit_user: >-
   {{ mssql_ad_kerberos_user }}@{{ ad_integration_realm | upper }}
+__mssql_configure_tuned: true
```

Comment:
Please move this line up in the file after `__mssql_conf_cli`, To keep it close with other general config related variables.

Reply:
i have changed it to __mssql_tuned_supported and made a new overwritable default mssql_manage_tuned, so that also redhat users may disable tuned, if they like.